src/cljs/villagebookUI/utils.cljs

```diff
@@ -22,5 +22,6 @@
            (store/add-user! res)
            (store/fetched!))
          (fn [res]
+           (store/add-user! nil)
```

Comment:
Avoid doing side effects directly in rendering functions. Don't couple your side effects to your component lifecycle. Your component lifecycle is ideally something that you don't want to think about at all.